### PR TITLE
Update for me ;)

### PR DESCRIPTION
**Pull request by jholt1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,4 +34,5 @@ Adding the dependency:
 
 ## Special thank you
 
-I've developed most of this extension by editing a very popular extension project called [github-mention-highlighter](https://github.com/benbalter/github-mention-highlighter) by GitHub user [benbalter](https://github.com/benbalter). Most of the code was re-used, including this very same README :smile:
\ No newline at end of file
+I've developed most of this extension by editing a very popular extension project called [github-mention-highlighter](https://github.com/benbalter/github-mention-highlighter) by GitHub user [jholt1](https://github.com/jholt1). Most of the code was re-used, including this very same README :smile:
```
Comment (bitoiu) [inline README.md:37]: Start typing and start review.

Comment (bitoiu) [reply README.md:37]: @jholt1 no notification sent.

File: README.md [hunk 2]
```diff
@@ -34,4 +34,5 @@ Adding the dependency:
 
 ## Special thank you
```
Comment (bitoiu) [inline README.md:35]: No changes needed

**Review by bitoiu — CHANGES_REQUESTED** [review]
Sorry @jholt1 can't remove the kudos from Ben :cry:

**Comment by jholt1** [comment]
Closing, thanks for the demo! @bitoiu

**Comment by bitoiu** [comment]
For testing purposes.

**Pull request closed without merging**